src/root/projects-show.js

```diff
@@ -28,7 +28,7 @@ const projectsShow = {
 
         return m('.project-show', {
             config: ctrl.setProjectPageTitle()
-        }, project() && (!ctrl.isSubscription(project()) || ctrl.subscriptionData()) ? [
```

Comment:
Are you sure we don't need this anymore? I was having some trouble with request loading order which left the goals box with blank data.

Reply:
locally it worked just fine. but if the problem persists, we should fix it by adding checks closer to the components that are rendering this data + controlling redrawing.